src/engine/bank03.asm

```diff
@@ -3478,10 +3527,38 @@ AfterTutorialBattleScript: ; d834 (3:5834)
         run_command ScriptCommand_CloseTextBox
         run_command ScriptCommand_PrintTextString
         tx Text05f2
-        run_command Func_d271
-; 0xd860
+.ows_d85f
+        run_command ScriptCommand_ShowMultichoiceTextbox_ChooseStarterDeck
+        run_command ScriptCommand_CloseTextBox
+        run_command ScriptCommand_AskQuestionJump
+        tx Text05f3
+        dw .ows_d869
+        run_command ScriptCommand_Jump
+        dw .ows_d85f
+; 0xd869
```

Comment:
I think without an incbin here there's no need for this label, but others may disagree (also this is mostly nitpicky)

Reply:
You mean address comment, right? Yeah, I typically delete the trailing address comment unless it is followed by in `INCROM` although this is an unwritten rule so I don't worry about it.